api/src/controllers/courtCases.ts

```diff
@@ -0,0 +1,20 @@
+import type { Response } from "express"
+import type { DataSource } from "typeorm"
+import getDataSource from "../services/getDataSource"
+import listCourtCases from "../services/listCourtCases"
+import type { CaseListQueryRequest } from "../../src/types/CaseListQueryRequest"
+
+export const getCourtCases = async (req: CaseListQueryRequest, res: Response) => {
```

Comment:
Nit: the file name does not match the function name

Reply:
My intention here was to have all of the controllers related to courtCases in this file.